### PR TITLE
Nrai/button and links stylings on agency updates/cc 41

### PR DESCRIPTION
**Pull request by nrailsy97**
- Changed button name from `Save` to `Submit`
- Changed link name from `Update` to `Submit Change`
- Redirected to a new page with message: `Organization update successfully submitted. An administrator will review your request.` after a change request is submitted

Screen Shot 2021-08-31 at 10 16 14 AM

Screen Shot 2021-08-31 at 10 19 30 AM

Screen Shot 2021-08-31 at 10 20 25 AM

Thank you @micronix !

### CODE REVIEW

File: app/views/agency_update_requests/edit.html.erb [hunk 1]
```diff
@@ -100,8 +100,7 @@
               </div>
             </div>
             <div class="panel-inline-body">
-              <%= link_to (t'buttons_t.back'), agency_update_requests_path, class: 'btn btn-primary' %>
-              <%= f.button (t'buttons_t.save'), class: "btn btn-info" %>
+              <%= link_to (t'buttons_t.back'), agency_update_requests_path, class: 'btn btn-info' %>
```
Comment (micronix) [inline app/views/agency_update_requests/edit.html.erb:103]: There should still be a "Save" button when an admin wants to save the submission but not approve it or reject it yet.

File: app/views/pages/confirmation.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  <%= render partial: "app/#{ENV['APP_NAME']}/header_container" %>
+</section>
+
+<div style="height: 250px">
+  <h4 style="background-color: lightyellow; padding:0.5em 1em;">Organization update successfully submitted. An administrator will review your request.</h4>
+</div>
+
+<div>
+  <p>Please, click <%= link_to "here", root_path %> to go to the homepage!</p>
+</div>
```
Comment (micronix) [inline app/views/pages/confirmation.html.erb:11]: This is not necessary we have the link at the top of the page to go back to the home page. Let's try to keep it as minimal as possible

File: config/locales/en.yml [hunk 3]
```diff
@@ -195,7 +195,7 @@ en:
     back: Back
     edit: Edit
     delete: Delete
-    save: Save
+    submit: Submit
```
Comment (micronix) [inline config/locales/en.yml:198]: we need both save and submit

File: config/locales/es.yml [hunk 4]
```diff
@@ -174,7 +174,7 @@ es:
     back: Atras
     edit: Actualizar
     delete: Eliminar
-    save: Guardar
+    submit: Enviar
```
Comment (micronix) [inline config/locales/es.yml:177]: need both save and submit

**Review by micronix — CHANGES_REQUESTED** [review]
just a few small changes

File: app/views/agency_update_requests/edit.html.erb (hunk 1 above)
Comment (nrailsy97) [reply app/views/agency_update_requests/edit.html.erb:103]: Oh, that makes sense. Thanks @micronix !

File: app/views/pages/confirmation.html.erb (hunk 2 above)
Comment (nrailsy97) [reply app/views/pages/confirmation.html.erb:11]: Sure, I will remove it!

**Review by micronix — APPROVED** [review]
Looks good

**Pull request merged**